tmk_core/common/eeconfig.c

```diff
@@ -50,6 +50,13 @@ void eeconfig_init_quantum(void) {
   eeprom_update_dword(EECONFIG_RGB_MATRIX,    0);
   eeprom_update_byte(EECONFIG_RGB_MATRIX_SPEED, 0);
 
+  // TODO: Remove once ARM has a way to configure EECONFIG_HANDEDNESS
```

Comment:
An alternative solution is to remove the logic from eeprom init, and relocate it to `is_keyboard_left` within `split_util.c`.